modules/mesh-layers/src/scenegraph-layer/scenegraph-layer.js

```diff
@@ -28,6 +29,7 @@ const vs = `
   attribute vec3 instancePositions;
   attribute vec2 instancePositions64xy;
   attribute vec3 instancePickingColors;
+  attribute mat4 instanceModelMatrix;
```

Comment:
@tsherif What do you think about using a `mat4` declaration for the attribute in the shader?

It looks a little cleaner this way, are there any caveats?

Reply:
I hope it is also faster this way because the compiler knows to keep all 16 values together so it can potentially optimize.

Reply:
No caveats, as long as the attribute pointers are set up correctly. No performance wins either though, since layout is exactly the same.